Lib/ufo2ft/filters/transformations.py

```diff
@@ -164,20 +162,16 @@ def filter(self, glyph, isComponent=False):
                 modified.add(base_name)
 
         if not isComponent:
-            for attr in ["height", "width",
-                         "leftMargin", "rightMargin",
-                         "topMargin", "bottomMargin",
-                         "verticalOrigin"]:
-                if attr in metrics:
-                    value = getattr(glyph, attr)
-                    if value is not None:
-                        value += metrics.get(attr)
-                        setattr(glyph, attr, value)
-                    else:
-                        logger.warning(
-                            "Cannot add %i to undefined %s in %s",
-                            metrics.get(attr), attr, glyph.name
-                        )
+            for attr, value in metrics.items():
+                current_value = getattr(glyph, attr)
+                if current_value is not None:
+                    value += current_value
+                    setattr(glyph, attr, value)
+                else:
+                    logger.warning(
+                        "Cannot add %i to undefined %s in %s",
+                        value, attr, glyph.name
```

Comment:
minor suggestion: you don’t need to increment value +=current_value, just setattr(glyph, attr, current_value+value)